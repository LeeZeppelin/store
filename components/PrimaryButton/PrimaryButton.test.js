import React from 'react';

import { render } from 'react-testing-library';
import { PrimaryButton } from '.';

describe('PrimaryButton', () => {
  it('renders correctly', () => {
    const { asFragment } = render(
      <PrimaryButton>With some text</PrimaryButton>
    );
    expect(asFragment()).toMatchSnapshot();
  });

  it('passes in children', () => {
    const { getByText } = render(<PrimaryButton>With some text</PrimaryButton>);
    expect(getByText('With some text')).toBeInTheDocument();
  });
});
