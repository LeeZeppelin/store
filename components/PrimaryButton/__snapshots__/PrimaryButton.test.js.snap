// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrimaryButton renders correctly 1`] = `
<DocumentFragment>
  <button
    class="PrimaryButton-vlil7y-0 hhriad"
  >
    With some text
  </button>
</DocumentFragment>
`;
